solc/CommandLineInterface.cpp

```diff
@@ -1152,6 +1218,55 @@ void CommandLineInterface::assembleYul(yul::YulStack::Language _language, yul::Y
 			else
 				serr() << "No text representation found." << std::endl;
 		}
+
+		if (m_options.compiler.outputs.asmJson)
```

Comment:
Oh, so this PR also adds support for `--asm-json` output in the assembler mode? This is a hidden feature not mentioned in the changelog :) And it should really be a separate commit (a separate PR even).

Reply:
Yes. I dropped the support here and move to another branch. I will open the PR for that shortly.